test/test-cli.sh

```diff
@@ -3,8 +3,16 @@
 set -e
 
 : ${IMAGE:?Need to provide environment variable IMAGE}
+: ${EXPECTED_ARCH:?Need to provide variable EXPECTED_ARCH}
 : ${EXPECTED_VERSION:?Need to provide environment variable EXPECTED_VERSION}
 
+IMAGE_ARCH=$(docker inspect ${IMAGE} | jq -r ".[].Architecture")
```

Comment:
Hmm,  are we using jq anywhere else or we need to install it and update developer docs?

Reply:
Where are these tests run? I tested it locally using `make test-cli` directly but that might not be exactly the same as where these tests actually run. I couldn't find it anywhere within the GitHub Actions.

Reply:
Yea, we are using Kubernetes own CI called Prow.

config
https://github.com/kubernetes/test-infra/blob/9e779315af77740c7bd1df0dc0b22d70f3a46b72/config/jobs/kubernetes-sigs/metrics-server/metrics-server-presubmits.yaml